src/util/mask.js

```diff
@@ -90,14 +90,15 @@ const maskValidates = (mask, char) => {
  * @return {String}
```

Comment:
This docblock needs updating if you're returning arbitrary values. Does jsdoc have generics?

Reply:
You know better than I do.

Reply:
http://usejsdoc.org/tags-param.html#multiple-types-and-repeatable-parameters